src/FeedsMigrateImporterInterface.php

```diff
@@ -19,4 +19,142 @@ interface FeedsMigrateImporterInterface extends ConfigEntityInterface {
    */
   const SCHEDULE_CONTINUOUSLY = 0;
 
+  /**
+   * Indicates that existing items should be left alone.
+   */
+  const EXISTING_LEAVE = 'leave';
+
+  /**
+   * Indicates that existing items should be replaced.
+   */
+  const EXISTING_REPLACE = 'replace';
```

Comment:
I am happy to remove this. I saw this was included in the original `feeds_migrate` code and just refactored the entire importer form/classes. I don't quite get the different either.

Reply:
Replace == wipe out all field values from an existing entity and let the source dictate what its new values should be.
The option erroneously no longer does that in Feeds 7.x-2.x and 8.x-3.x, probably because folks forgot what it was supposed to do when porting Feeds to D7 and D8 (including me, until I found that D6 issue: https://www.drupal.org/project/feeds/issues/753426).